deluge/plugins/WebUi/deluge_webui/core.py

```diff
@@ -30,17 +30,28 @@
 
 log = logging.getLogger(__name__)
 
-DEFAULT_PREFS = {'enabled': False, 'ssl': False, 'port': 8112}
+DEFAULT_PREFS = {
+    'enabled': False,
+    'ssl': False,
+    'webapidoc_enabled': False,
```

Comment:
* Do we need a config to enable/disable docs? 
* If so should we enable it by default? Users will never know about it and like me annoyed when they need to set a config ;)
* And rename this setting to just *docs*?

Reply:
I'm not sure. You think it should always be available? You'll have to log in to do anything of importance anyways, so maybe we can drop the setting.